src/stores/GroupStoreCache.js

```diff
@@ -18,18 +18,20 @@ import GroupStore from './GroupStore';
 
 class GroupStoreCache {
     constructor() {
-        this.groupStores = {};
+        this.groupStore = null;
     }
 
     getGroupStore(matrixClient, groupId) {
-        if (!this.groupStores[groupId]) {
-            this.groupStores[groupId] = new GroupStore(matrixClient, groupId);
+        if (!this.groupStore || this.groupStore._groupId !== groupId) {
```

Comment:
Looks like you removed the underscore, so just `groupId`?

Reply:
oops